fn/client/src/reducers/Cart-reducer.js

```diff
@@ -7,22 +7,22 @@ const userId = getFromLocalStorage('userId')
 const productCollection = getFromLocalStorage('products_collection')
 const localCartNumbers = getFromLocalStorage('cart_numbers');
 
-const saveCart = async (userId, cart) => {
-  axios.put(`https://lv487node-backend.herokuapp.com/users/cart/${userId}`, { cart });
-}
+const saveCart = async (userId, data, token) => {
+  return axios({ method: 'PUT', url: `http://localhost:5000/users/cart/${userId}`, data , headers: { "x-auth-token": token } })
```

Comment:
Hard code! Please take it to config: 
export const base_URL = `http://localhost:5000/ in config file
import in component and return axios({ method: 'PUT', url: `${base_URL}/users/cart/${userId}`, data , headers: { "x-auth-token": token } })